src/ASM/LR/ASMLRSpline.C

```diff
@@ -11,14 +11,15 @@
 //!
 //==============================================================================
 
-#include "ASMunstruct.h"
-#include "IFEM.h"
 #include "LRSpline/LRSplineSurface.h"
 #include "LRSpline/Basisfunction.h"
+
+#include "ASMLRSpline.h"
```

Comment:
should be first include.

Reply:
Gues that is only cosmetic? And since all other ASM-classes have the GoTools/LRSpline headers first, I wanted to be consistent with that convention. But I can swap if you insist.

Reply:
well, it's a knee-jerk more than anything, to avoid missing dependencies in the interface. but if it is like this everywhere, i don't insist.